src/main/java/fr/foodlimit/api/security/MockSecurityConfig.java

```diff
@@ -0,0 +1,48 @@
+package fr.foodlimit.api.security;
+
+import fr.foodlimit.api.security.jwt.JWTConfigurer;
+import fr.foodlimit.api.security.jwt.TokenProvider;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+
+@Profile("test")
+@Configuration
+public class MockSecurityConfig extends WebSecurityConfigurerAdapter {
+
+  private final TokenProvider tokenProvider;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "tokenProvider" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)